test/test-all.js

```diff
@@ -17,21 +17,12 @@ BPromise.config({
 const app = createApp();
 
 function getPdfTextContent(buffer) {
-  return new BPromise((resolve, reject) => {
-    const pdfParser = new PDFParser();
-    pdfParser.on('pdfParser_dataError', (err) => {
-      reject(err);
-    });
-    pdfParser.on('pdfParser_dataReady', () => {
-      resolve(pdfParser.getRawTextContent());
-    });
-
-    pdfParser.parseBuffer(buffer);
-  });
+  // Fix for characters encoding issues
+  return pdf(buffer).then(data => data.text.replace(/[\W_]+/g, '-'));
```

Comment:
Replace all non-alphanumeric characters with a hyphen to resolve some difference in character encoding when comparing strings extracted from the PDF and strings defined in the test environment